net/shadowsocks-libev/Makefile

```diff
@@ -16,9 +16,12 @@ PKG_NAME:=shadowsocks-libev
 PKG_VERSION:=3.3.5
 PKG_RELEASE:=7
```

Comment:
No version bump? No version change?  I don't think that the approach, which you chose is a good.

Reply:
What would be a good approach?